types.go

```diff
@@ -0,0 +1,7 @@
+package k8sclient
+
+import "k8s.io/apimachinery/pkg/runtime"
+
+// SchemeBuilder is an optional way to extend the known types to the global
+// client-go scheme. Make use of it for custom CRs.
```

Comment:
Maybe adding some example on how to setup this thing makes sense.

```suggestion
// client-go scheme. Make use of it for custom CRs.
//
// Typical usage:
//
//	k8sclient.SchemeBuilder{
//		myapiversion.AddToShcheme,
//	}
//
```